Homework_14/mammals.py

```diff
@@ -0,0 +1,17 @@
+from abc import abstractmethod
+
+from animals import Animal
+
+
+class Mammal(Animal):
```

Comment:
If you inherited from abstract class you should implements eat() and move() methods